cpp/src/aztec/proof_system/flavor/flavor.hpp

```diff
@@ -144,6 +144,13 @@ struct StandardHonk {
             gemini_evaluation_entries.emplace_back(transcript::Manifest::ManifestEntry(
             { .name = "a_" + std::to_string(i), .num_bytes = fr_size, .derived_by_verifier = false }));
         };
+        gemini_evaluation_entries.emplace_back(transcript::Manifest::ManifestEntry(
+            { .name = "a_0_pos", .num_bytes = fr_size, .derived_by_verifier = false }));
+        // Include two additional commitments that depend on challenge "r" from previous round
+        gemini_evaluation_entries.emplace_back(transcript::Manifest::ManifestEntry(
+              { .name = "FOLD_0_pos", .num_bytes = g1_size, .derived_by_verifier = false }));
+        gemini_evaluation_entries.emplace_back(transcript::Manifest::ManifestEntry(
+              { .name = "FOLD_0_neg", .num_bytes = g1_size, .derived_by_verifier = false }));
         manifest_rounds.emplace_back(transcript::Manifest::RoundManifest(
             gemini_evaluation_entries,
```

Comment:
Can we call these something like "folding entries"? The name suggests it's all evaluations, obscuring the fact that there are two commitments in there.

Reply:
Yeah I toyed with some different names but couldn't decide on anything better. The previous round name is `fold_commitment_entries` so would need to change that one as well for clarity? I ultimately decided `gemini_evaluation_entries` makes decent sense because the round consists of evaluations and special partially evaluated commitments. Let me know what you prefer and I'll change it accordingly.

Reply:
OK, I see the conflict with the previous round. I think we should come up with better names for both, but I guess now is not the time.